reactjs/src/stores/tokenAuthStore.ts

```diff
@@ -0,0 +1,85 @@
+import { observable, action, computed } from 'mobx';
+import tokenAuthService from 'src/services/tokenAuth/tokenAuthService';
+import { AuthenticationResultModel } from './../services/tokenAuth/dto/authenticationResultModel';
+
+const tokenStorageName = 'abp.AuthToken';
+
+class TokenAuthStore {
+  @observable authenticateResult: AuthenticationResultModel | null;
+
+  constructor() {
+    //Kullanıcı herhangi bir nedende dolayı sayfadan cıkışşsa
+    //storagedan tokenı tekrar stora almayı sağlar
```

Comment:
Could you write comments in English? And I think there are unnecessary comments. For example: `// clear local storage`, ` //false ise login olunmamıştır` and etc. Could you please remove these unnecessary comments?

Reply:
Resolved